### ISSUE TITLE
spawner should create dynamic vats, rather than evaluating code in the spawner's own vat

### ISSUE THREAD

**Issue by warner** [Agoric/agoric-sdk#1343]
We should move the `spawner` package to put code into new dynamic vats, just like the new split-Zoe does.

This will unblock #1341.

**Comment by warner** [comment]
I think @Chris-Hibbert suggested that this might be in his arena, although much lower priority than everything else right now.

**Comment by Chris-Hibbert** [comment]
What I hadn't realized is that the spawner is being used from dapp deploy scripts to kick off the persistent portion of the dapp. We need to find a different solutions for those uses (hopefully that relies on dynamic vats) so we can drop the spawner, since the spawner continues to rely on some metering code we'd like to expunge.

**Comment by warner** [comment]
I'm taking this one back over, because I want to remove within-vat metering right away. From @katelynsills and @Chris-Hibbert 's advice, I'm going to strip out all of the remaining zoe-like features (contract identity, invitations, access to source code, etc). It looks like all of the dapps that use the spawner are not using those features.
